maintainers/maintainer-list.nix

```diff
@@ -4337,6 +4337,15 @@
     githubId = 405105;
     name = "Dustin Frisch";
   };
+  foolnotion = {
```

Comment:
Now squash the commits, creating only two:
- one for maintainers list (keep it)
- other for the package itself